### DISCUSSION TITLE
Cabelos

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#319] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/cabelos/

**Comment by one-eyed-willy** [comment]
Se uma mulher disse, então deve ser verdade

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/cabelos/#comment_1129022043>):

> Se uma mulher disse, então deve ser verdade

E como você pode ter certeza de que sou mulher?

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/cabelos/#comment_1129042205>):

> E como você pode ter certeza de que sou mulher?

Verdade. Ela pode ser um cara de 130kg que vive no porão da mãe no subúrbio de Detroit chamado Chuck :stuck\_out\_tongue:

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/cabelos/#comment_1129043945>):

> Verdade. Ela pode ser um cara de 130kg que vive no porão da mãe no subúrbio de Detroit chamado Chuck 😛

![null](<https://i.postimg.cc/Pr70MG7M/bugs-life-male-ladybug-scene.png>)
